packages/core/nuxt-theme-module/theme/components/WishlistSidebar.vue

```diff
@@ -100,25 +120,50 @@ export default {
     SfCollectedProduct,
     SfImage
   },
-  setup() {
+  setup(props, { root }) {
     const { isWishlistSidebarOpen, toggleWishlistSidebar } = useUiState();
-    const { wishlist, removeItem, load: loadWishlist } = useWishlist();
+    const { wishlist, removeItem: removeFromWishlist, load: loadWishlist, clear: clearWishlist } = useWishlist();
     const { isAuthenticated } = useUser();
+    const { addItem } = useCart();
     const products = computed(() => wishlistGetters.getItems(wishlist.value));
     const totals = computed(() => wishlistGetters.getTotals(wishlist.value));
     const totalItems = computed(() => wishlistGetters.getTotalItems(wishlist.value));
 
+    watch(() => root.$route, () => {
+      if (isWishlistSidebarOpen.value) toggleWishlistSidebar();
+    });
+
+    const addToCart = async (product) => {
+      await addItem({
+        product: { sku: product.variant.sku },
+        quantity: 1
```

Comment:
Should this be hardcoded? I don't remember exactly whether we have a qty ref in wishlist but we could introduce it here to add item with desired quantity, not just one.

Reply:
@Baroshem Unfortunately not, there is only a button `Add to cart`, without the qty selector
![image](https://user-images.githubusercontent.com/39009379/128703170-d4e539d5-ffbb-48c3-b5c6-337bf86e0e4b.png)

Reply:
Ok, so let's leave it as it is :)